packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -253,23 +276,28 @@ class RetainingPathWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return ValueListenableBuilder<RetainingPath?>(
       valueListenable: retainingPath,
-      builder: (context, retainingPath, child) {
+      builder: (context, retainingPath, _) {
+        final retainingObjects = retainingPath == null
+            ? <Widget>[]
+            : _retainingPathList(
+                context,
+                retainingPath,
+              );
         return VmExpansionTile(
           title: 'Retaining Path',
           onExpanded: onExpanded,
           children: retainingPath == null
```

Comment:
This can be rewritten to be a bit nicer:

```dart
children: [
  if (retainingPath == null)
    SizedCircularProgressIndicator()
  else
    SizedBox.fromSize(
      ...
    ),
],
```

Reply:
Thanks